cloud/logging.rst

```diff
@@ -26,34 +26,108 @@ The Citus Cloud dashboard automatically shows you the most recent 100 log lines
 External Log Destinations
 -------------------------
 
-For anything more than a quick look at your logs, we recommend setting up an external provider as a log destination. Through this method you'll receive
-all logs using the Syslog protocol, and can analyze and retain them according to your own preferences.
+For anything more than a quick look at your logs, we recommend setting up an external provider. Through this method the logs will transmit to a dedicated logging service and you can analyze and retain them according to your own preferences. To use an external provider, create a new logging destination in the Citus Cloud console. For instance, here is the new destination dialog filled in with Papertrail settings:
```

Comment:
The previous content notes the Syslog protocol in a few different places. Has this changed or less significant now, leading us to remove its mention?